src/items/projectile_manager.cpp

```diff
@@ -153,3 +153,32 @@ bool ProjectileManager::projectileIsClose(const AbstractKart * const kart,
     }
     return false;
 }   // projectileIsClose
+
+// -----------------------------------------------------------------------------
+/** Returns an int containing the numbers of a given flyable in a given radius
+ *  around the kart
+ *  \param kart The kart for which the test is done.
+ *  \param radius Distance within which the projectile must be.
+ *  \param type The type of projectile checked
+*/
+int ProjectileManager::projectileCloseType(const AbstractKart * const kart,
```

Comment:
Very minor, but 'projectileCloseType' doesn't sound like correct english to me. GetNearbyProjectilesCount maybe ?

Reply:
Done